systemtest/src/test/java/io/strimzi/systemtest/LogSettingST.java

```diff
@@ -290,6 +301,193 @@ void testKubectlGetStrimzi() {
+                            .endContainer()
+                        .endSpec()
+                    .endTemplate()
+                .endSpec()
+                .done();
+
+        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
+
+            kafka.getSpec().getKafka().getJvmOptions().setGcLoggingEnabled(false);
+            kafka.getSpec().getZookeeper().getJvmOptions().setGcLoggingEnabled(false);
+            kafka.getSpec().getEntityOperator().getUserOperator().getJvmOptions().setGcLoggingEnabled(false);
+            kafka.getSpec().getEntityOperator().getTopicOperator().getJvmOptions().setGcLoggingEnabled(false);
+
+            kafka.getSpec().getKafka()
+                    .setLogging(new ExternalLoggingBuilder()
+                        .withName(configMapKafkaName).build());
+
+            kafka.getSpec().getZookeeper().setLogging(new ExternalLoggingBuilder()
+                    .withName(configMapZookeeperName)
+                    .build());
+
+            kafka.getSpec().getEntityOperator().getTopicOperator().setLogging(new ExternalLoggingBuilder()
+                    .withName(configMapOpName)
+                    .build());
+
+            kafka.getSpec().getEntityOperator().getUserOperator().setLogging(new ExternalLoggingBuilder()
+                    .withName(configMapOpName)
+                    .build());
+
+        });
+
+        zkPods = StatefulSetUtils.waitTillSsHasRolled(KafkaResources.zookeeperStatefulSetName(CLUSTER_NAME), 1, zkPods);
+        kafkaPods = StatefulSetUtils.waitTillSsHasRolled(KafkaResources.kafkaStatefulSetName(CLUSTER_NAME), 3, kafkaPods);
+        eoPods = DeploymentUtils.waitTillDepHasRolled(KafkaResources.entityOperatorDeploymentName(CLUSTER_NAME), 1, eoPods);
+
+        TestUtils.waitFor("Logs in CO", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT, () -> {
+            String clusterOperatorName = kubeClient().listPodsByPrefixInName("strimzi-cluster-operator").get(0).getMetadata().getName();
+            String logs = "{" + kubeClient().logs(clusterOperatorName).split("\\+(.*?)\\n\\{")[1];
+            try {
+                new JsonObject(logs);
+                LOGGER.info("JSON format logging successfully set for {}", clusterOperatorName);
+                return true;
+            } catch (Exception e) {
+                LOGGER.info("Failed to set JSON format logging for {}", clusterOperatorName);
+                return false;
+            }
+        });
+
+        assertThat(checkLogForJSONFormat(kafkaPods, "kafka"), is(true));
+        assertThat(checkLogForJSONFormat(zkPods, "zookeeper"), is(true));
+        assertThat(checkLogForJSONFormat(eoPods, "topic-operator"), is(true));
+        assertThat(checkLogForJSONFormat(eoPods, "user-operator"), is(true));
+    }
+
+    private boolean checkLogForJSONFormat(Map<String, String> pods, String containerName) {
```

Comment:
Maybe move to some utils since there is not assert? But I will leave it on you

Reply:
I will move it to utils :)

Reply:
Done :)